src/meta/processors/BaseProcessor.h

```diff
@@ -141,7 +142,7 @@ class BaseProcessor {
     /**
      * General put function.
      * */
-    void doPut(std::vector<kvstore::KV> data);
+    void doPut(std::vector<kvstore::KV> data, bool refresh = true);
```

Comment:
IIRC,  for balance procedure,  it will not call doPut interface inside BaseProcessor.

And in most cases, we don't want the lastUpdateTime being updated.  

So we'd better control it by each processor not in the BaseProcessor.

Reply:
done!